lib/OpenQA/WebAPI/Controller/API/V1/Feature.pm

```diff
@@ -0,0 +1,21 @@
+package OpenQA::WebAPI::Controller::API::V1::Feature;
+use strict;
+use warnings;
+use Mojo::Base 'Mojolicious::Controller';
+
+sub informed {
+    my ($self)  = @_;
+    my $user    = $self->current_user;
+    my $version = $self->param('version');
+    $user->update({last_login_version => $version});
+    my $seen = $self->param('seen');
```

Comment:
what else would seen be? You want to the user to confirm he didn't see it? :)

IMO these 2 fields are duplicating each other. Assume I visited openqa last on version 3 and now it's 5. So I need 2 features to be presented. Once these are presented, they are gone - and should never appear again. So the last_login_version is enough to know the features to be presented.